### ISSUE TITLE
No implementation of the proposed 'TeCoA' found in the code.

### ISSUE THREAD

**Issue by LiQiiiii** [cvlab-columbia/ZSRobust4FoundationModel#7]
Hi there, it seems that the loss function used in the code is typically cross entropy, I didn't found the implementation of the proposed TeCoA, but TeCoA is indeed the biggest novelty of this work.
